shard.yml

```diff
@@ -7,6 +7,7 @@ authors:
 
 dependencies:
   amq-protocol:
+    #path: ../amq-protocol.cr
```

Comment:
```suggestion
```
A tip here is to use `shard.override.yml` (which should be added to .gitignore)

Reply:
thanks! didn't know of it